extension/background/language/langs/english.toml

```diff
@@ -18,6 +18,8 @@ haste = "paste"
 taste = "paste"
 pace = "paste"
 best = "paste"
+"copy links" = "copy"
```

Comment:
"copy links" doesn't _always_ mean exactly the same as "copy". E.g., "copy links title" shouldn't mean the same thing as "copy title" (I don't know what it should mean... but it doesn't make much sense).

So instead this should be changed in `clipboard.toml` so that the existing `copy link` phrase also matches `copy links`